src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -289,6 +290,16 @@ public string GetLanguage()
         }
 
         /// <summary>
+        /// Package License Url
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Remove ``<returns></returns>``  It's always better to just drop empty triple slash comment tags.